samples/custom-login/README.md

```diff
@@ -0,0 +1,24 @@
+# custom-login sample
```

Comment:
"custom login" and "uses widget" don't line up for me.  

I'd expect:
* "self-hosted widget" - self describing
* "custom login" - doesn't use the widget, collects credentials manually

Reply:
I personally find `self-hosted`, `custom login`, `okta hosted` pretty confusion to me. As I saw in react-samples, custom login actually equal to `self-hosted widget`, how about let's get the terms aligned internally and then only use the agreed terms.

Reply:
I was following the convention in the samples (samples-js-react, samples-js-angular), but maybe the samples are a bad pattern. What is called "custom login" in the samples is actually "self hosted widget", it is not a true "custom" login flow (which could show how to implement MFA without SIW)

Reply:
I think we're agreed that the current terms are confusing.  I back Shuo's idea to just agree on better terms and this is a good place to start using them.

How about:
* okta-hosted
* customized-okta-hosted (I doubt this will be something we sample, but this sounds like the term to use if we do)
* self-hosted-widget
* custom-login

?  (I'm not sold on these, happy to see alternatives)

Reply:
I am just wondering how many places are using these `terms`? samples, docs, ios/android? should we involve any other teams in the discussion to get the terms aligned?

Reply:
The samples are now generated from templates so it is trivial to rename them. These are the current names:
- static-spa
- webpack-spa
- express-web-no-oidc
- express-web-with-oidc

Both spa applications support redirect, self-hosted widget, and custom form (authn) as the "flow" option. The main difference between the apps is webpack vs. static files.

The express web (no oidc) is a minimal sample. The express web (with oidc) adds the oidc feature. These could work as steps in a tutorial.